Resources/doc/reference/troubleshooting.rst

```diff
@@ -106,3 +106,27 @@ Finally your settings in your ``sonata_media`` parameters will look like this:
             local:
                 directory:  "%kernel.root_dir%/../web/uploads/media"
                 create:     false
+
+ 
+Error on cache:warmup
```

Comment:
I would name this section "Usage without the admin bundle" (this might not be the last issue).